netcdf_to_tplot.py

```diff
@@ -0,0 +1,156 @@
+
+    Returns:
+        List of tplot variables created.
+
+    Examples:
+        >>> #Create tplot variables from a GOES netCDF file
+        >>> import pytplot
+        >>> file = "/Users/user_name/goes_files/g15_epead_a16ew_1m_20171201_20171231.nc"
+        >>> pytplot.netcdf_to_tplot(file, prefix='mvn_')
+
+        >>> #Add a prefix, and plot immediately.
+        >>> import pytplot
+        >>> file = "/Users/user_name/goes_files/g15_epead_a16ew_1m_20171201_20171231.nc"
+        >>> pytplot.netcdf_to_tplot(file, prefix='goes_prefix_', plot=True)
+
+    '''
+
+    stored_variables = []
+    global data_quants
+
+    print(data_quants)
+
+    if isinstance(filenames, str):
+        filenames = [filenames]
+    elif isinstance(filenames, list):
+        filenames = filenames
+    else:
+        print("Invalid filenames input.")
+        #return stored_variables
+
+    for filename in filenames:
+        file = Dataset(filename, "r+")
+        vars_and_atts = {} # Dictionary that will contain variables and their attributes (in case this info needs to be queried)
+        for name, variable in file.variables.items():
+            vars_and_atts[name] = {}
+            for attrname in variable.ncattrs():
+                vars_and_atts[name][attrname] = getattr(variable, attrname)
+
+        masked_vars = {} # Dictionary containing properly masked variables
+        for var in vars_and_atts.keys():
+            # Grabbing each variable and replacing missing values with np.nan (if not already np.nan)
+            reg_var = file.variables[var]
+            try:
+                var_fill_value = vars_and_atts[var]['missing_value']
+                if np.isnan(var_fill_value) != True:
+                    # We want to force missing values to be nan so that plots don't look strange
+                    var_mask = np.ma.masked_where(reg_var==np.float32(var_fill_value),reg_var)
+                    var_filled = np.ma.filled(var_mask,np.nan)
+                elif np.isnan(var_fill_value) == True:
+                    # missing values are already np.nan, don't need to do anything
+                    var_filled = reg_var
+            except: #continue # Go to next iteration, this variable doesn't have data to mask (probably just a descriptor variable (i.e., 'base_time')
+                var_filled = reg_var
+            masked_vars[var] = var_filled
+
+        # Most files are from GOES data, which seems to usually have 'time_tag' in them that contain time information.
+        # There is an exception filter below that will allow a user to pick a different time variable if time_tag doesn't exist.
+        if 'time_tag' in vars_and_atts.keys():
+            time_var = file['time_tag']
+            unix_times = change_time_to_unix_time(time_var)
+
+        if 'time_tag' not in vars_and_atts.keys():
+            time_var_exists = input('Is there a differently-named time variable in the variables (y/n)? \nVariable list: {l}'.format(l=vars_and_atts.keys()))
```

Comment:
This is great, but can you also add a variable in the function call where someone can specify the name of the time variable there?  

My thinking is if someone comes along with a bunch of netCDF files where the time variables are all named the something weird, they probably don't want to have to go through this process each time.  So they have the option of saying something like " time_var_name='whatever' " when they call the function.